src/libostree/ostree-repo-pull.c

```diff
@@ -64,6 +64,7 @@ typedef struct {
   GPtrArray     *meta_mirrorlist;    /* List of base URIs for fetching metadata */
   GPtrArray     *content_mirrorlist; /* List of base URIs for fetching content */
   OstreeRepo   *remote_repo_local;
+  GPtrArray    *repo_refs; /* Array<OstreeRepo> */
```

Comment:
A bit bikeshed, but can we call this the full `repo_references`? (Or maybe even `repo_local_references`). Given that "refs" already means something else in this code.